explorer/api.py

```diff
@@ -19,6 +19,7 @@ def title_or_slug(item):
     return item.get("slug", "")
 
 
+@app.route("/robots.txt")
```

Comment:
this is a decorator for the `main` function, so I'd expect it to still serve the compare page/main page at `robots.txt`.

Should this use something like this? http://flask.pocoo.org/docs/0.12/api/#flask.send_from_directory

I'm assuming there's not a simple way to make heroku serve this?